packages/shared/lib/typings/migration.ts

```diff
@@ -224,7 +224,7 @@ export function getMigrationAddress(
     bridge: Bridge,
     __ids: CommunicationIds,
     ledgerPrompt: boolean,
-    accountIndex: number
+    accountIndex: string | number
```

Comment:
Should we rename this parameter to `accountId` rather than `accountIndex`?

Reply:
Yes, good catch @maxwellmattryan.

Reply:
Fixed in f3b9dad

Reply:
Nice. I should've mentioned that there's also this [`AccountIdentifier`](https://github.com/iotaledger/firefly/blob/develop/packages/shared/lib/typings/account.ts#L43) type in `typings/account.ts` we can use here!